pkg/clients/secretversion/secretversion.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package secretversion
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/crossplane/provider-gcp/apis/secretversion/v1alpha1"
+	"github.com/google/go-cmp/cmp"
+	"github.com/googleapis/gax-go"
+	"google.golang.org/genproto/googleapis/cloud/secretmanager/v1"
```

Comment:
I think both `Secret` and `SecretVersion` should be in `secretmanager` group, just be different types within it.

Reply:
Done!